pages/linux/alien.md

```diff
@@ -1,6 +1,7 @@
 # alien
 
 > Convert different installation packages to other formats.
+> See also: `debtap`, for .deb conversion on Arch Linux.
```

Comment:
Maybe we can update it a bit:

```suggestion
> See also: `debtap`, for DEB conversion on Arch Linux.
```
Or

```suggestion
> See also: `debtap`, for `.deb` conversion on Arch Linux.
```

Reply:
> Maybe we can update it a bit:
> ```suggestion
> > See also: `debtap`, for DEB conversion on Arch Linux.
> ```

The official documentation doesn't use the term "DEB package", it only uses the term "Debian package", so the 2 possibilities are Debian package or `.deb`. I think `.deb` is clearer because of Debian-derivates e.g. Ubuntu.

Reply:
We have file extensions in caps in many other pages. Thats why I think that DEB fits best.

Reply:
Well, unlike RPM for example, DEB is not an acronym (and doesn't even exist), then it's wrong. So I think we should prefer `.deb` over DEB.